source/index.html.md.erb

```diff
@@ -14538,7 +14524,7 @@ emails = response.data
 
 `GET /users/{userId}/alternateemails`
 
-Gets a list of the Alternate Emails for the specified User. 
+Gets a summarized list of the Alternate Emails for the specified User. 
```

Comment:
Not full objects?

Reply:
Hmm. There are only 3 things in an object, but I can't test this. I just get a data[] back when I try to test it on myself. Do you have a sample user I could use?

Reply:
Tested. You do get the full object back. I'll rewrite.

Reply:
Done.